### ISSUE TITLE
Crash when using Cockpit Terminal

### ISSUE THREAD

**Issue by SysAdminShed** [AlmaLinux/almalinux-deploy#205]
The almalinux-deploy script does crash when being executed via the Cockpit Terminal.

Since Cockpit gets reinstalled for Alma Linux 8, it does reset the connection and will crash the script to migrate from Rocky 8 to Alma 8.

Tested in Dev twice and replicated twice.

If you reboot the device, you simply get in rescue directly since it's missing almost everything and GRUB2 completely.

**Comment by codyro** [comment]
I'm not sure if this is something that can easily be worked around--ideally you're running the script via SSH or via the console.

**Comment by andrewlukoshko** [comment]
I think we just need to update Migration guide to mention using reliable console access.
@sboldyreva

**Comment by SysAdminShed** [comment]
I completely agree, I just think this would need to be mentioned in case someone tries it like I did.

**Comment by yuravk** [comment]
Corresponded warning added to the Wiki.
